example/integration_test/shelley_address_test_cases.dart

```diff
@@ -476,7 +474,7 @@ List<ShelleyTestCase> shelleyTestCases = [
   ),
   ShelleyTestCase(
     testName: 'reward script 2',
-    network: CardanoNetwork.custom(customNetworkId: 0x03, customNetworkMagic: 47, customBech32Hrp: 'addr'),
+    network: CardanoNetwork.custom(customNetworkId: 0x03, customNetworkMagic: 47, customBech32Hrp: 'stake'),
```

Comment:
Cred ca ar trebui sa le separam in `customAddrBech32Hrp` si `customStakeBech32Hrp`

Reply:
P.s. O sa trebuiasca modificat (posibil) incat SDK-ul sa foloseasca rewards hrp cand address params este reward (key sau script)

Reply:
In versiunea actuala, nu se intampla sa ai nevoie de paymentBech32Hrp si stakeBech32Hrp in acelasi timp pe ledger. Asa ca in SDK nu e nevoie sa le setezi pe amandoua simultan, pentru ca nu folosim ambele deodata.